docs/features/sessions/changing-instance-type.md

```diff
@@ -0,0 +1,25 @@
+---
+description: Run (and sweep) any private or public Github repository.
+---
+import Note from "@site/src/components/Note";
+
+# Change Instance Type of a Session
+
+```text
+grid session change-instance-type [--spot , --on_demand] SESSION_NAME g4dn.xlarge
+```
+
+Changing Sessison instance type allows you to upgrade or downgrade the compute capacity of the Session while keeping all of your work in progress untouched.
+
+![](/images/sessions/change-instance-type.gif)
+
+You can change the instance type of the session when it is PAUSED
```

Comment:
Test both CLI and UI to see if instance type can be changed while Session is not paused. 
Update this line with that information.

Reply:
We can change instance when the session is running, but we need to pause and resume for instance.. i could test that i changed from CPU to CPU, but GPU to GPU is not working [https://linear.app/gridai/issue/GRID-8622/change-instance-type-from-gpu-to-gpu-does-not-change-the-instance](url)